fiber/adapter.go

```diff
@@ -134,7 +145,7 @@ func (f *FiberLambda) adaptor(w http.ResponseWriter, r *http.Request) {
 
 	// Set response headers
 	fctx.Response.Header.VisitAll(func(k, v []byte) {
-		w.Header().Set(utils.UnsafeString(k), utils.UnsafeString(v))
```

Comment:
You can see this Issue.

- https://github.com/gofiber/adaptor/issues/64
- https://github.com/gofiber/adaptor/commit/9b07027c9d0df28b92029831716e141b34203ce0